vyper/venom/passes/normalization.py

```diff
@@ -19,72 +19,45 @@ def _split_basic_block(self, bb: IRBasicBlock) -> None:
             jump_inst = in_bb.instructions[-1]
             assert bb in in_bb.cfg_out
 
-            # Handle static and dynamic branching
-            if jump_inst.opcode == "jnz":
-                self._split_for_static_branch(bb, in_bb)
-            elif jump_inst.opcode == "jmp" and isinstance(jump_inst.operands[0], IRVariable):
-                self._split_for_dynamic_branch(bb, in_bb)
-            else:
-                continue
-
-            self.changes += 1
-
-    def _split_for_static_branch(self, bb: IRBasicBlock, in_bb: IRBasicBlock) -> None:
-        jump_inst = in_bb.instructions[-1]
-        for i, op in enumerate(jump_inst.operands):
-            if op == bb.label:
-                edge = i
+            # Handle branching
+            if jump_inst.opcode == "jnz" or (
```

Comment:
I know we hardcode the opcodes in a lot of places, but it would be actually very nice to have them constants, also giving us the opportunity to have an overview of used opcodes and a place for documentation